rotkehlchen/assets/resolver.py

```diff
@@ -70,10 +75,21 @@ def resolve_asset(
             return cached_data
 
         # If was not found in the cache try querying it in the globaldb
-        asset = GlobalDBHandler().resolve_asset(
-            identifier=identifier,
-            form_with_incomplete_data=form_with_incomplete_data,
-        )
+        try:
+            asset = GlobalDBHandler().resolve_asset(
+                identifier=identifier,
+                form_with_incomplete_data=form_with_incomplete_data,
+            )
+        except UnknownAsset:
```

Comment:
Didn't I already write it guys? I wrote it before, please for the love of god read my comments and also understand our code.

https://github.com/rotki/rotki/pull/5063#discussion_r1009401454

> It should only happen when the asset is in the list of constant assets and an unknown asset or a wrong asset type exception is raised

You are ignoring an entire category of possible errors here

```suggestion
        except (UnknownAsset, WrongAssetType):
```

Reply:
`resolve_asset` can only raise `WrongAssetType` if the user went to the database and modified the column value for type unless I'm wrong https://github.com/rotki/rotki/blob/0fa9a7cc2dee688db0621750a31b2c6c49c31108/rotkehlchen/globaldb/handler.py#L1770

Reply:
`GlobalDBHandler().resolve_asset()` cannot raise `WrongAssetType`. The only place that error is raised is in 

https://github.com/rotki/rotki/blob/0fa9a7cc2dee688db0621750a31b2c6c49c31108/rotkehlchen/assets/resolver.py#L113